crates/paralegal-policy/src/flows_to.rs

```diff
@@ -11,7 +11,48 @@ use crate::NodeType;
 /// The transitive closure of the [`Ctrl::data_flow`] relation.
 ///
 /// Implemented efficiently using an [`IndexedDomain`] over the
-/// [`DataSource`] and [`DataSink`] types.
+/// [`DataSource`] and [`CallSiteOrDataSink`] types.
+///
+/// Relationship of [`CtrlFlowsTo::data_flows_to`], [`CtrlFlowsTo::flows_to`],
```

Comment:
I would make this an actual heading (prefix with `##`)

Reply:
done